test/reader/handlers_test.clj

```diff
@@ -0,0 +1,31 @@
+(ns reader.handlers-test
+  (:require [clojure.test :refer :all]
+            [reader.handlers :as h]
+            [reader.html :as html]
+            [reader.redis :as r]
+            [reader.test-utils :as tu]
+            [reader.fixtures :as f]))
+
+(deftest fetch-page-test
+  (testing "Given url query parameter, return status 200 response"
+    (with-redefs [html/new-html (constantly "hello world")]
+      (let [request {:query-params {"url" "foo.bar"}}
+            response (h/fetch-page request)]
+        (is (= 200 (:status response))))))
+  (testing "Given request without url parameter, return status 400 response"
```

Comment:
**[NIT]** Would be good to separate testing blocks with newlines for readability.